misc/config_tools/launch_config/pt.py

```diff
@@ -136,7 +136,7 @@ def audio_pt(user_vm_type, sel, vmid, config):
 
     if bdf_audio:
         print("kernel_version=$(uname -r)", file=config)
-        print('audio_module="/usr/lib/modules/$kernel_version/kernel/sound/soc/intel/boards/snd-soc-sst_bxt_sos_tdf8532.ko"', file=config)
+        print('audio_module="/usr/lib/modules/$kernel_version/kernel/sound/soc/intel/boards/snd-soc-sst_bxt_xxx_tdf8532.ko"', file=config)
```

Comment:
Valid kernel module filename?

Reply:
@gvancuts  Have  already delete the kernel module for specific boards.